grakn-test/src/main/java/ai/grakn/test/GraknTestEnv.java

```diff
@@ -141,6 +160,15 @@ static void stopEngine(GraknEngineServer server) throws Exception {
         // There is no way to stop the embedded Casssandra, no such API offered.
     }
 
+    static Service startSpark(GraknEngineConfig config) {
+        LOG.info("starting spark on port " + config.uri());
+
+        Service spark = Service.ignite();
+        configureSpark(spark, config, JWTHandler.create(config.getProperty(JWT_SECRET_PROPERTY)));
```

Comment:
Yes. This is [an existing bug](https://work.grakn.ai/entity/15230), so I'm happy to just leave it for now.